projects/demo-playwright/tests/kit/input-date-range/input-date-range.spec.ts

```diff
@@ -129,5 +134,28 @@ test.describe(`InputDateRange`, () => {
                 );
             });
         });
+
+        test(`Select from [items] => select date range from calendar`, async ({page}) => {
+            const calendar = new TuiCalendarPO(
+                inputDateRange.calendarRange.locator('tui-calendar'),
+            );
+
+            await tuiGoto(
+                page,
+                `components/input-date-range/API?items$=1&sandboxExpanded=true`,
+            );
+
+            await inputDateRange.textfield.click();
+            await inputDateRange.selectItem(1);
+            await expect(inputDateRange.textfield).toHaveValue('Today');
+
+            await inputDateRange.textfield.click();
+            await calendar.clickOnCalendarDay(21);
+
+            await expect(inputDateRange.textfield).toHaveValue('21.09.2020 – 25.09.2020');
+            await expect(example).toHaveScreenshot(
+                '07-item-and-calendar-interactions.png',
```

Comment:
**Before fix:**
![07-item-and-calendar-interactions-actual](https://github.com/taiga-family/taiga-ui/assets/35179038/9c9fac4c-c8b9-4fa9-8e4a-89d703bc4889)

Reply:
**After fix:**
![07-item-and-calendar-interactions-expected](https://github.com/taiga-family/taiga-ui/assets/35179038/81af8d87-800f-46f7-aed5-bac3bb5e8715)